kernel-module/build-for-jetson-drive-igpu-native.sh

```diff
@@ -20,22 +20,22 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-KERNEL_SRC_DIR="/usr/src/nvidia/nvidia-oot/"
-if [ ! -d ${KERNEL_SRC_DIR} ]; then
-	echo "Kernel source not found in $KERNEL_SRC_DIR"
-	KERNEL_SRC_DIR="/usr/src/./linux-headers-tegra-oot*"
-	if [ ! -d ${KERNEL_SRC_DIR} ]; then
-		echo "Kernel source not found in $KERNEL_SRC_DIR"
+NVIDIA_OOT_SRC_DIR="/usr/src/nvidia/nvidia-oot/"
+if [ ! -d ${NVIDIA_OOT_SRC_DIR} ]; then
+	echo "Nvdia OOT source not found in $NVIDIA_OOT_SRC_DIR"
+	NVIDIA_OOT_SRC_DIR="/usr/src/./linux-headers-tegra-oot*"
+	if [ ! -d ${NVIDIA_OOT_SRC_DIR} ]; then
+		echo "Nvidia OOT  source not found in $NVIDIA_OOT_SRC_DIR"
```

Comment:
Only have echo "Nvidia OOT source not found"

Reply:
Updated debug message. 👍